### DISCUSSION TITLE
Trying to model cam and cable

### DISCUSSION THREAD

**Discussion by dummifiedme** [getfem-doc/getfem#59] [Q&A]
I wished to model Cam and Cable, is it possible with this package?

![3c20f293-c142-474e-8f56-4fc21b80f953](https://user-images.githubusercontent.com/68785729/182979530-857a2f87-bbc7-4461-b08a-93671a6af52c.jpeg)

PS. I don't know if this is the right place to ask a question, I didn't find any discussion/forum.

**Comment by tkoyama010** [comment]
Thank you for asking a question. Could you send an email to getfem-users@nongnu.org. It is the most active discussion place.
